pkg/service/tools/coverage.go

```diff
@@ -0,0 +1,349 @@
+package tools
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"go.keploy.io/server/v2/pkg/models"
+)
+
+func CalGoCoverage() (models.TestCoverage, error) {
```

Comment:
why are these in tools service.?

Reply:
coverage calculation by analyzing the raw coverage files may be viewed as a tool, thats why I put it here, do you want to move it to a different place ?

Reply:
move it into utils in test service

Reply:
done!